lib/apr/payments.ex

```diff
@@ -34,6 +48,16 @@ defmodule Apr.Payments do
     end
   end
 
+  defp charge_data_ach(payment_intent) do
+    with [charge | _tail] <- payment_intent.charges.data do
+      %{
+        risk_level: charge.outcome.risk_level
```

Comment:
The risk level for ACH transactions from Stripe was recently supported and very basic. Let's start from here and see if other signals might be useful and iterate!